### PR TITLE
Add Acala and Bitcoin fork chains

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/verge/info/info.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Verge",
+    "website": "https://vergecurrency.com",
+    "description": "Verge is a privacy-focused cryptocurrency and blockchain that seeks to offer a fast, efficient, decentralized payments network that improves upon the original Bitcoin (BTC) blockchain.",
+    "explorer": "https://verge-blockchain.info",
+    "symbol": "XVG",
+    "type": "coin",
+    "decimals": 8,
```
Comment (satoshiotomakan) [inline blockchains/verge/info/info.json:8]: Verge seems to have 6 decimals: https://github.com/vergecurrency/verge/blob/b9a3f7278ba9c71e993decf38e5fcf731feb8c1d/src/qt/vergeunits.cpp#L89
https://github.com/trustwallet/wallet-core/blob/4443c337f6337aa7fcb76bceb3987ad6a1fbbaf2/registry.json#L444

**Review by satoshiotomakan — CHANGES_REQUESTED** [review]
Missing blockchains: [Komodo](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L1209), [IOST](https://github.com/trustwallet/wallet-core/blob/master/registry.json#L3487)

**Comment by satoshiotomakan** [comment]
Do we need to add [Scroll](https://github.com/trustwallet/wallet-core/pull/3179), [Conflux eSpace](https://github.com/trustwallet/wallet-core/pull/3259) too?

**Comment by Milerius** [comment]
> Do we need to add [Scroll](https://github.com/trustwallet/wallet-core/pull/3179), [Conflux eSpace](https://github.com/trustwallet/wallet-core/pull/3259) too? cc: @cryptomanz1 @Milerius

Yes

**Review by satoshiotomakan — APPROVED** [review]
Thank you for the changes! LGTM

**Comment by Setarland** [comment]
Blakchin

**Pull request merged**